Pear/Controller/LoginViewController.swift

```diff
@@ -101,41 +101,14 @@ class LoginViewController: UIViewController {
             UserDefaults.standard.set(userEmail[..<addressSignIndex], forKey: Constants.UserDefaults.userNetId)
```

Comment:
shouldn't we cache user information after successfully authenticating?

Reply:
we should prob do this rip. ill make task for this